runtime/doc/lua.txt

```diff
@@ -253,6 +253,11 @@ arguments separated by " " (space) instead of "\t" (tab).
         :lua print(_VERSION)
 <    To see the LuaJIT version: >vim
         :lua =jit.version
+
+:{range}lua
+    Executes the |[range]| in the current buffer as Lua code. Unlike |:source|,
+    this will execute the specified lines regardless of the extension or
+    |'filetype'| of the buffer.
 <
```

Comment:
Remove this `<`

Reply:
Oh oops, not remove but place this block below it right?

Reply:
Either works.